src/main/java/org/opensearch/sdk/ExtensionNamedWriteableRegistry.java

```diff
@@ -98,14 +100,15 @@ public NamedWriteableRegistryResponse handleNamedWriteableRegistryRequest(OpenSe
      * @throws IOException if InputStream generated from the byte array is unsuccessfully closed
      * @return A response acknowledging the request to parse has executed successfully
      */
+    @SuppressWarnings("unchecked")
```

Comment:
Similar to the comment on the companion PR, move this down to the local variable at line 111.

Reply:
For this `@SuppressWarnings`, if I move this down to the local variable at line 111, it will cause error like `extensionEntries cannot be resolved to a type`. That's the reason that I move `@SuppressWarnings` to the top of method.

Reply:
I think that's because of the change on the OpenSearch side, if you publish to maven local it should work.